app/angular/src/server/angular-cli_config.js

```diff
@@ -133,6 +134,11 @@ export function applyAngularCliWebpackConfig(baseConfig, cliWebpackConfigOptions
     modules: Array.from(
       new Set([...baseConfig.resolve.modules, ...cliCommonConfig.resolve.modules])
     ),
+    plugins: [
+      new TsconfigPathsPlugin({
+        configFile: cliWebpackConfigOptions.buildOptions.tsConfig,
```

Comment:
so should we still add the "cliCommonConfig.resolve.modules" ? can you fix then the hardcoded `'src/tsconfig.app.json'` up there ?

Reply:
See my latest commit - Hope I changed the line you meant